.github/workflows/ci.yaml

```diff
@@ -0,0 +1,86 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
```

Comment:
Do you need master?

Reply:
For Bors the "trying" branch is usually enough. Perhaps we should add master too though.